containers/codespaces-linux/.devcontainer/Dockerfile

```diff
@@ -150,6 +150,9 @@ RUN apt-get update -yq\
     && install -o codespace -g codespace -m 755 /tmp/codespace/git-ed.sh /home/codespace/.local/bin/git-ed.sh \
     && sudo -u codespace git config --global core.editor "/home/codespace/.local/bin/git-ed.sh" \
     #
+    # Alias code-insiders to code for easy terminal file editing
```

Comment:
Would it cause problems if a user connects from both at different points in time?

Reply:
I don't think so, since the VS Code remote server sets itself on the PATH as part of spin up, so I'd expect each instance of VS Code's extension host terminals to only have itself on the PATH. But it's not a scenario I've actually tested.

Reply:
I think this should work, so I'm going to go ahead and leave it. If we find out it causes any problems for some edge-case scenarios, we can revisit.